nginx.location.conf

```diff
@@ -0,0 +1,12 @@
+location /berlin {
+  alias /srv/sommer17-webseite/public;
+  index index.html;
+  try_files $uri @sommer17-gunicorn;
```

Comment:
Würde das in ein seperates location auslagern:

location /berlin/newsletter/newsletter-subscribe {
          try_files $uri @sommer17-gunicorn;
      }